### PR TITLE
Add version cli argument

### LINKED ISSUE #1651: Add --version cli argument

### PR DESCRIPTION
**Pull request by papr**
Fixes #1651

### CODE REVIEW

File: pupil_src/main.py [hunk 1]
```diff
@@ -18,9 +18,23 @@
...
+from version_utils import get_version
+
+app_version = get_version()
+if parsed_args.version:
+    running_from = "bundle" if running_from_bundle else "source"
+    version_message = textwrap.dedent(
+        f"""
+        Pupil {parsed_args.app.capitalize()} version {app_version} ({running_from})
+        """
+    ).strip()
```
Comment (pfaion) [inline pupil_src/main.py]: @papr Are you importing textwrap and doing this multiline-string plus `.strip()` only to keep the line-length?
Why not
```python
version_message = (
    f"Pupil {parsed_args.app.capitalize()} version {app_version} ({running_from})"
)
```
Or are there any other things happening that I don't spot currently?
Or if it's still too long:
```python
version_message = (
    f"Pupil {parsed_args.app.capitalize()}"
    f"version {app_version} ({running_from})"
)
```
And now that I think more about it, why not just:
```python
print(
    f"Pupil {parsed_args.app.capitalize()}"
    f"version {app_version} ({running_from})"
)

Comment (papr) [reply pupil_src/main.py]: Not sure what I was thinking.

File: pupil_src/main.py [hunk 2]
```diff
@@ -18,9 +18,22 @@
...
+from version_utils import get_version
+
+app_version = get_version()
+if parsed_args.version:
+    running_from = "bundle" if running_from_bundle else "source"
+    version_message = (
+        f"Pupil {parsed_args.app.capitalize()} version {app_version} ({running_from})"
+    )
+
+    print(version_message)
```
Comment (pfaion) [inline pupil_src/main.py]: shouldn't we `logger.info` this instead of `print`?

Comment (papr) [reply pupil_src/main.py]: The loggers are not setup yet at this point.

**Pull request merged**